src/lightkurve/interact.py

```diff
@@ -750,10 +753,11 @@ def _create_lightcurve_from_pixels(
             tpf, selected_pixel_indices, transform_func=transform_func
         ):
             """Create the lightcurve from the selected pixel index list"""
+            tpf.flux[0].shape
             selected_indices = np.array(selected_pixel_indices)
-            selected_mask = np.isin(pixel_index_array, selected_indices)
-            lc_new = tpf.to_lightcurve(aperture_mask=selected_mask)
-            lc_new.meta["APERTURE_MASK"] = selected_mask
+            selected_mask = np.isin(pixel_index_array.reshape(-1), selected_indices.reshape(-1))
```

Comment:
1. `pixel_index_array` and `selected_indices` are already in 1d. The additional reshape is unncessary.
2. We might want to rename `selected_mask` to `selected_mask_1d`, so make it clear this is a 1d version. Or we could convert it to 2D right here.